constraint/pkg/client/drivers/k8scel/driver.go

```diff
@@ -1,29 +1,41 @@
 package k8scel
 
-// import (
-// 	"context"
-// 	"errors"
-// 	"fmt"
-// 	"strings"
-// 	"sync"
-
-// 	apiconstraints "github.com/open-policy-agent/frameworks/constraint/pkg/apis/constraints"
-// 	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers"
-// 	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
-// 	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
-// 	"github.com/open-policy-agent/opa/storage"
-// 	admissionv1alpha1 "k8s.io/api/admissionregistration/v1alpha1"
-// 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
-// 	"k8s.io/apimachinery/pkg/runtime"
-// 	"k8s.io/apimachinery/pkg/runtime/schema"
-// 	"k8s.io/apiserver/pkg/admission"
-// 	"k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy"
-// 	"k8s.io/apiserver/pkg/admission/plugin/webhook/generic"
-// )
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+	"time"
+
+	apiconstraints "github.com/open-policy-agent/frameworks/constraint/pkg/apis/constraints"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/client/drivers"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/core/templates"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/instrumentation"
+	"github.com/open-policy-agent/frameworks/constraint/pkg/types"
+	"github.com/open-policy-agent/opa/storage"
+	admissionv1 "k8s.io/api/admission/v1"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	admissionv1alpha1 "k8s.io/api/admissionregistration/v1alpha1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/admission/plugin/cel"
+	"k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy"
+	"k8s.io/apiserver/pkg/admission/plugin/webhook/matchconditions"
+	auditinternal "k8s.io/apiserver/pkg/apis/audit"
+	celAPI "k8s.io/apiserver/pkg/apis/cel"
+	"k8s.io/apiserver/pkg/authentication/user"
+)
+
+// NOTE: This is a PROTOTYPE driver. Do not use this for any critical work
```

Comment:
maybe each driver should have a field for level of maturity so consumers can see/evaluate, instead of these comments that only maintainers see :) just a thought. not a PR blocker.

Reply:
Yeah, def. worth having more metadata around drivers (which version of CEL is it using)? For now this is just to warn any potential linkers of the code that "thar be dragons"